config/locales/en.yml

```diff
@@ -20,4 +20,4 @@
 # available at http://guides.rubyonrails.org/i18n.html.
 
 en:
-  hello: "Hello world"
+  hello: "Hello world"
```

Comment:
i was suppose to work in this en.yml file in other branch but mistakenly done some changes on this branch so i just put it to default